packages/@aws-cdk/toolkit-lib/lib/actions/refactor/index.ts

```diff
@@ -14,4 +14,18 @@ export interface RefactorOptions {
    * @default - all stacks
    */
   stacks?: StackSelector;
+
+  /**
+   * The absolute path to a file that contains a list of resources to
+   * skip during the refactor. The file should be in JSON format and
+   * contain an array of _destination_ locations that should be skipped,
+   * i.e., the location to which a resource would be moved if the
+   * refactor were to happen.
```

Comment:
Why is _destination_ the better choice over _source_? My gut reaction was: "huh, but how do I know what the destination is?" Is this a concern?

Reply:
To be clear: I genuinely don't know the answer here. But if there's two choices we always run the risk of picking the less natural one.

Reply:
Okay, so the thinking here is: The code base (and therefore the output assembly) have already been changed by the user. Refactoring is matching the new target state with the previous state from CFN. We are addressing resources by what they are currently called in the users local code base?

I can dig that. We might want to tweak the wording and explanation here a bit (maybe AI can help?), but otherwise all good.